packages/components/src/button/index.tsx

```diff
@@ -49,6 +49,11 @@ function useDeprecatedProps( {
 	};
 
 	if ( isSmall ) {
+		deprecated( 'Button isSmall prop', {
+			since: '6.6',
+			alternative: 'size="small"',
+			version: '6.9',
```

Comment:
```suggestion
```

I'm thinking we don't really have to do a hard deprecation on this one, since the maintenance cost is very low. What do you think?

Reply:
I think we should start following those more actively and actually clean them all up when they're due. With that in mind, I find them useful and I think they help reduce technical debt over time.